common/src/chain/config.rs

```diff
@@ -192,6 +194,37 @@ pub fn create_mainnet() -> ChainConfig {
     }
 }
 
+pub fn create_regtest() -> ChainConfig {
+    let chain_type = ChainType::Regtest;
+    let pow_config = PoWChainConfig::new(chain_type);
+
+    let upgrades = vec![
+        (
+            BlockHeight::new(0),
+            UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::IgnoreConsensus),
+        ),
+        (
+            BlockHeight::new(1),
+            UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::PoW {
+                initial_difficulty: pow_config.limit().into(),
+            }),
+        ),
+    ];
+
+    ChainConfig {
```

Comment:
Is there a specific reason the data in here is the mainnet but the fn is called regtest? Should `address_prefix` not be `REGTEST_ADDRESS_PREFIX` for example?

Reply:
It should be, I don't know how I missed that. But apparently I don't know the alphabets either so there's that.

Reply:
A typo by Aaro. I'll see if I can fix it. I think it won't break anything.